packages/synapse-react-client/src/components/row_renderers/utils/CardFooter.tsx

```diff
@@ -47,35 +53,30 @@ class CardFooter extends Component<CardFooterProps, State> {
   }
 
   renderRowValue = (
-    columnName: string,
-    value: string,
-    tableColumnName: string,
+    key: string,
+    value: React.ReactNode,
+    tableColumnName?: string,
   ) => {
     const columnIconOptions = this.props.columnIconOptions
-    if (!value.match || !value.trim) {
-      // value can sometimes be a react element, so it doesn't have a .match function, interestingly I didn't
-      // see typeof return 'object' for that case which would be a better check.
+    if (typeof value !== 'string') {
+      // value can sometimes be a react element
       return value
     }
-    value = value.trim()
-    const doiLink = convertDoiToLink(value)
+    const valueAsString = value.trim()
+    const doiLink = convertDoiToLink(valueAsString)
 
     if (doiLink) {
       return (
-        <a
-          data-search-handle={columnName}
```

Comment:
I removed all of the data-search-handle attributes -- they were added in 2020 and I cannot find any documentation (in the repo or on the internet) about what they are for